eng/scripts/Language-Settings.ps1

```diff
@@ -6,6 +6,7 @@ $BlobStorageUrl = "https://azuresdkdocs.blob.core.windows.net/%24web?restype=con
 
 function Get-python-PackageInfoFromRepo  ($pkgPath, $serviceDirectory, $pkgName)
 {
+  pip install packaging==20.4 -q -I
```

Comment:
Do we need the `-I`? Isn't that going to force a reinstall all the time?

Reply:
Yes the intention is to force reinstall it, otherwise it will tell you its already installed and am trying to avoid getting that message.